docs/build.md

```diff
@@ -0,0 +1,77 @@
+# Development Env
+
+The EasyReflectometry project consists of two packages; an application and a library. 
+To develop effectively for EasyReflectometry, we work on the `develop` branch of both and then periodically merge to `master` when a release is made. 
```

Comment:
The default branch is `main` not `master`

Reply:
```suggestion
To develop effectively for EasyReflectometry, we work on the `develop` branch of both and then periodically merge to `main` when a release is made. 
```